src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/SmoModel/TreeNodeDefinition.xml

```diff
@@ -308,6 +324,14 @@
       <Filter Property="IsSystemObject" Value="0" Type="bool" />
     </Filters>
   </Node>
+  <Node Name="SystemTableValuedFunctions" LocLabel="SR.SchemaHierarchy_TableValuedFunctions" BaseClass="ModelBased" Strategy="MultipleElementsOfType" ChildQuerierTypes="SqlUserDefinedFunction" TreeNode="TableValuedFunctionTreeNode">
+    <Filters >
+      <Filter Property="FunctionType" Type="Enum" ValidFor="Sql2016|SqlvNext|AzureV12">
```

Comment:
All these function types work on SQL 2005+ as far as I know. For both system and non-system folders, I think we need to fix so we no longer filter to just SQL2016 and Azure. Did I mess this up in the original metadata?